FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/CreateMessageFragment.kt

```diff
@@ -195,6 +195,11 @@ class CreateMessageFragment : BaseSyncFragment(), View.OnFocusChangeListener,
     }
   }
 
+  override fun onDestroyView() {
+    super.onDestroyView()
```

Comment:
If in onCreate() the sequence is such we first run superclass stuff then local, then, logically, destruction should do things in the opposite way. But here I see that `super.onDestroyView()` is called first, then some local cleanup is performed. Is it correct? If correct then why?

Reply:
:) It refers to the Fragments lifecycle. Usually, in overridden methods, we should call the super methods first to perform the basic logic that is described in the superclass. Sorry for the short answer but it's a little difficult to explain how Android works in a few lines :)

Reply:
You can read more info here https://developer.android.com/guide/components/activities/activity-lifecycle